src/components/views/game/gamesummary/WinnersPodium.tsx

```diff
@@ -4,7 +4,7 @@ import { useCurrentGame } from "../../../../hooks/api/useCurrentGame"
 import trophy from "../../../../img/winnerwinnerchickendinner.png"
 import lost from "../../../../img/betterluck.png"
 
-import "./WinnersPodium.scss"
+import "../gamesummary/GameSummary.scss"
```

Comment:
?
WinnersPodium should use WinndersPodium.scss, and not share the css with another component